exchanges/bithumb/bithumb_test.go

```diff
@@ -62,91 +68,110 @@ func TestMain(m *testing.M) {
+	require.NoError(t, err, "GetTicker must not error")
+	assert.Positive(t, tick.OpeningPrice, "OpeningPrice should be positive")
+	assert.Positive(t, tick.ClosingPrice, "ClosingPrice should be positive")
+	assert.Positive(t, tick.MinPrice, "MinPrice should be positive")
+	assert.Positive(t, tick.MaxPrice, "MaxPrice should be positive")
+	assert.Positive(t, tick.UnitsTraded, "UnitsTraded should be positive")
+	assert.Positive(t, tick.AccumulatedTradeValue, "AccumulatedTradeValue should be positive")
+	assert.Positive(t, tick.PreviousClosingPrice, "PreviousClosingPrice should be positive")
+	assert.Positive(t, tick.UnitsTraded24Hr, "UnitsTraded24Hr should be positive")
+	assert.Positive(t, tick.AccumulatedTradeValue24hr, "AccumulatedTradeValue24hr should be positive")
+	assert.NotEmpty(t, tick.Fluctuate24Hr, "Fluctuate24Hr should not be empty")
+	assert.NotEmpty(t, tick.FluctuateRate24hr, "FluctuateRate24hr should not be empty")
+	assert.Positive(t, tick.Date, "Date should be positive")
+}
+
+// not all currencies have dates and fluctuation rates
 func TestGetAllTickers(t *testing.T) {
 	t.Parallel()
-	_, err := b.GetAllTickers(context.Background())
-	if err != nil {
-		t.Error(err)
+	tick, err := b.GetAllTickers(context.Background())
+	require.NoError(t, err, "GetAllTickers must not error")
+	for _, res := range tick {
+		assert.Positive(t, res.OpeningPrice, "OpeningPrice should be positive")
+		assert.Positive(t, res.ClosingPrice, "ClosingPrice should be positive")
+		assert.Positive(t, res.MinPrice, "MinPrice should be positive")
+		assert.Positive(t, res.MaxPrice, "MaxPrice should be positive")
+		assert.Positive(t, res.UnitsTraded, "UnitsTraded should be positive")
+		assert.Positive(t, res.AccumulatedTradeValue, "AccumulatedTradeValue should be positive")
+		assert.Positive(t, res.PreviousClosingPrice, "PreviousClosingPrice should be positive")
+		assert.Positive(t, res.UnitsTraded24Hr, "UnitsTraded24Hr should be positive")
+		assert.Positive(t, res.AccumulatedTradeValue24hr, "AccumulatedTradeValue24hr should be positive")
 	}
 }
 
 func TestGetOrderBook(t *testing.T) {
 	t.Parallel()
-	_, err := b.GetOrderBook(context.Background(), testCurrency)
-	if err != nil {
-		t.Error(err)
+	ob, err := b.GetOrderBook(context.Background(), testCurrency)
+	require.NoError(t, err, "GetOrderBook must not error")
+	assert.NotEmpty(t, ob.Status, "Status should not be empty")
+	assert.NotEmpty(t, ob.Data.Timestamp, "Timestamp should not be empty")
+	assert.NotEmpty(t, ob.Data.OrderCurrency, "OrderCurrency should not be empty")
+	assert.NotEmpty(t, ob.Data.PaymentCurrency, "PaymentCurrency should not be empty")
+	for _, a := range ob.Data.Asks {
+		assert.Positive(t, a.Price, "Price should be positive")
+		assert.Positive(t, a.Quantity, "Quantity should be positive")
+	}
+	for _, b := range ob.Data.Bids {
+		assert.Positive(t, b.Price, "Price should be positive")
+		assert.Positive(t, b.Quantity, "Quantity should be positive")
```

Comment:
Main reason I'm against this kind of testing is that it suggests that we have caused an error, when its actually something that Bithumb is doing wrong. In an ideal situation, this test would always work. But then we get results like this:
`{"price":"99959000","quantity":"0"}`
So please remove these finer-grain `assert.Positive` tests. We validate orderbook data later

Full response:
```
Bithumb raw response: {"status":"0000","data":{"timestamp":"1711406102914","payment_currency":"KRW","order_currency":"BTC","bids":[{"price":"99959000","quantity":"0"},{"price":"99922000","quantity":"0.1425"},{"price":"99921000","quantity":"0.101"},{"price":"99920000","quantity":"0.0001"},{"price":"99919000","quantity":"0.0026"},{"price":"99918000","quantity":"0.4919"},{"price":"99914000","quantity":"0.045"},{"price":"99913000","quantity":"0.0001"},{"price":"99912000","quantity":"0.0111"},{"price":"99908000","quantity":"0.0102"},{"price":"99907000","quantity":"0.0509"},{"price":"99906000","quantity":"0.0001"},{"price":"99900000","quantity":"0.0716"},{"price":"99899000","quantity":"0.0014"},{"price":"99898000","quantity":"0.7362"},{"price":"99896000","quantity":"0.7362"},{"price":"99895000","quantity":"0.0001"},{"price":"99894000","quantity":"0.0001"},{"price":"99893000","quantity":"0.0002"},{"price":"99892000","quantity":"0.0001"},{"price":"99891000","quantity":"0.0001"},{"price":"99890000","quantity":"0.0001"},{"price":"99889000","quantity":"0.0001"},{"price":"99888000","quantity":"0.0002"},{"price":"99887000","quantity":"0.0001"},{"price":"99886000","quantity":"0.0001"},{"price":"99875000","quantity":"0.0554"},{"price":"99874000","quantity":"0.4733"},{"price":"99861000","quantity":"0.445"},{"price":"99849000","quantity":"0.0002"}],"asks":[{"price":"99960000","quantity":"0.0001"},{"price":"99979000","quantity":"0.0002"},{"price":"99980000","quantity":"0.0006"},{"price":"99986000","quantity":"0.0002"},{"price":"99988000","quantity":"0.0043"},{"price":"99998000","quantity":"0.024"},{"price":"99999000","quantity":"0.0541"},{"price":"100000000","quantity":"0.681"},{"price":"100001000","quantity":"0.0372"},{"price":"100004000","quantity":"0.0358"},{"price":"100005000","quantity":"0.2469"},{"price":"100007000","quantity":"0.0216"},{"price":"100008000","quantity":"0.0247"},{"price":"100010000","quantity":"0.0011"},{"price":"100015000","quantity":"0.0029"},{"price":"100018000","quantity":"0.0011"},{"price":"100020000","quantity":"0.0001"},{"price":"100025000","quantity":"0.0001"},{"price":"100026000","quantity":"0.0007"},{"price":"100030000","quantity":"0.0007"},{"price":"100031000","quantity":"0.0028"},{"price":"100032000","quantity":"0.3854"},{"price":"100035000","quantity":"0.0551"},{"price":"100038000","quantity":"0.368"},{"price":"100039000","quantity":"0.0007"},{"price":"100040000","quantity":"0.0002"},{"price":"100046000","quantity":"0.0003"},{"price":"100048000","quantity":"0.0001"},{"price":"100049000","quantity":"0.0002"},{"price":"100050000","quantity":"0.0109"}]}}
--- FAIL: TestGetOrderBook (0.22s)
    bithumb_test.go:125: 
        	Error Trace:	/Users/wif420/go/src/github.com/thrasher-corp/gocryptotrader/exchanges/bithumb/bithumb_test.go:125
```

Reply:
noted and removed 👍🏼